code/modules/emotes/emote_sounds.dm

```diff
@@ -0,0 +1,52 @@
+/*
+* Declare for storing species and gender specific emote sounds and make them overridable between species. 
+* Also the soundlist is loaded from a json array that sorts lists of sounds by emote name and optionally gender.
+*/
+
+//Mainly used for debugging and verifying everything loaded properly
+var/global/list/decl/emote_sounds/all_emote_decl = list()
```

Comment:
This is redundant - `get_decls_of_type(/decl/emote_sounds)` returns a list of all emote decls.

Reply:
Yeah, I'm aware. Its because I couldn't get the global decl list table to display in the global variables while debugging, so I left that here for that reason. Otherwise its nearly impossible to see the state of the list.
I'll remove it once debugging is done.